ms2query/networking.py

```diff
@@ -0,0 +1,72 @@
+import networkx as nx
+# from networkx.drawing.nx_pydot import graphviz_layout
+import numpy as np
+
+
+def matches2network(query_id, matches):
+    """Return a networkx Graph connecting query_id to all matches
+
+    Args:
```

Comment:
```suggestion
    Parameters
```

Reply:
Not sure, but I thought that was how we had it in the first functions?
There are different styles and all are OK, only that we should be consistent. It will help to set up an automatic generation of a ``sphinx`` documentation based on those docstrings.

Reply:
We started out with Args, so I just continued with it. If you want we can change it to parameters of course.

Reply:
I don't have a strong preference. We can stick to Args: then!

Reply:
Cool! In your library_matching function you also used Args:, so that also seems to be your instinctive choice :p